rpc/types/src/other.rs

```diff
@@ -68,7 +68,8 @@ define_request_and_response! {
         key_images: Vec<String>,
     },
     AccessResponseBase {
-        spent_status: Vec<u8>, // TODO: should be `KeyImageSpentStatus`.
+        /// FIXME: These are [`KeyImageSpentStatus`] in [`u8`] form.
+        spent_status: Vec<u8>,
```

Comment:
For later: `cuprate_epee_encoding` doesn't blanket impl `Vec<impl EpeeValue>` so although this really should be `Vec<KeyImageSpentStatus>`, it's `Vec<u8>` for now.

Reply:
blanket impl `Vec<impl EpeeValue>` we can't do this until specialization btw.